src/lwc/geBatchGiftEntryTable/geBatchGiftEntryTable.js

```diff
@@ -125,6 +127,14 @@ export default class GeBatchGiftEntryTable extends LightningElement {
             .catch(error => handleError(error));
     }
 
+    retrieveBatchCurrencyIsoCode() {
+        retrieveBatchCurrencyIsoCode({batchId: this.batchId})
```

Comment:
Did you consider adding the batch ISO code to the data import model? Since the batch and its rows are being queried at initialization time already it might not be a bad idea to add this as an attribute to the model instead of a separate query.

Reply:
I decided to go the separate query route since we will be moving away from this data structure in the near future once we retire old BGE. I'm open to chatting about it though because I wrestled with the decision.